bin/sl-deploy.js

```diff
@@ -81,6 +87,9 @@ if (!serviceName) {
   process.exit(1);
 }
 
+clusterSize = (isNaN(clusterSize) || clusterSize < 1) ?
```

Comment:
This will set the cluster size to the number of cpus on the dev laptop. If --size isn't provided, don't set the deployment size, let the server decide.